.perltidyrc

```diff
@@ -0,0 +1,13 @@
+-pbp     # Start with Perl Best Practices
```

Comment:
`-pbp` without this line is user-hostile:

    -nst     # undo -st from -pbp, to allow for command line use